### PR TITLE
feat: [CET-1548] Update Error Tracking event list tab in CD to show e…

### PR DESCRIPTION
**Pull request by ricksjames**
…vents related to the CD service, environment, and executionId.

### Summary

Update Error Tracking event list tab in CD to show events related to the CD service, environment, and executionId.

#### Screenshots

Screenshot 2023-07-07 at 3 11 47 PM

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by ricksjames** [comment]
retrigger cypress-cv

**Review by abeg777 — DISMISSED** [review]

**Review by pankajsingh-harness — DISMISSED** [review]

File: src/modules/70-pipeline/pages/execution/ExecutionErrorTrackingView/ExecutionErrorTrackingView.tsx [hunk 1]
```diff
@@ -19,8 +19,12 @@ const ErrorTrackingApp = React.lazy(() => import('errortracking/App'))
 export default function ExecutionErrorTrackingView(): React.ReactElement | null {
   const context = useExecutionContext()
   const params = useParams<PipelineType<ExecutionPathProps>>()
+  const { module } = useParams<PipelineType<ModulePathParams>>()
   const history = useHistory()
   const pipelineExecutionDetail = context?.pipelineExecutionDetail
+  const cdStageInfo = pipelineExecutionDetail?.pipelineExecutionSummary?.moduleInfo?.cd || {}
+  const envIds = cdStageInfo?.envIdentifiers
+  const serviceIds = cdStageInfo?.serviceIdentifiers
```
Comment (rameshx) [inline src/modules/70-pipeline/pages/execution/ExecutionErrorTrackingView/ExecutionErrorTrackingView.tsx:27]: Nit: `?.` is not needed as `cdStageInfo` will be `{}` if `pipelineExecutionDetail?.pipelineExecutionSummary?.moduleInfo?.cd` is falsy

File: src/modules/70-pipeline/pages/execution/ExecutionErrorTrackingView/ExecutionErrorTrackingView.tsx [hunk 2]
```diff
@@ -38,23 +42,38 @@ export default function ExecutionErrorTrackingView(): React.ReactElement | null
...
-        versionId={pipelineExecutionDetail.pipelineExecutionSummary.runSequence.toString()}
-        fromDateTime={Math.floor(pipelineExecutionDetail.pipelineExecutionSummary.startTs / 1000)}
-        toDateTime={Math.floor(
-          pipelineExecutionDetail.pipelineExecutionSummary.endTs
-            ? pipelineExecutionDetail.pipelineExecutionSummary.endTs / 1000 + 60 * 30
-            : Date.now() / 1000
-        )}
-        handleBackAction={history.location.pathname.endsWith('/arc') ? handleBackAction : undefined}
-      />
+      {module == 'ci' ? (
```
Comment (rameshx) [inline src/modules/70-pipeline/pages/execution/ExecutionErrorTrackingView/ExecutionErrorTrackingView.tsx:45]: use `===`

**Review by rameshx — CHANGES_REQUESTED** [review]

File: src/modules/70-pipeline/pages/execution/ExecutionErrorTrackingView/ExecutionErrorTrackingView.tsx (hunk 1 above)
Comment (ricksjames) [reply src/modules/70-pipeline/pages/execution/ExecutionErrorTrackingView/ExecutionErrorTrackingView.tsx]: fixed.

File: src/modules/70-pipeline/pages/execution/ExecutionErrorTrackingView/ExecutionErrorTrackingView.tsx (hunk 2 above)
Comment (ricksjames) [reply src/modules/70-pipeline/pages/execution/ExecutionErrorTrackingView/ExecutionErrorTrackingView.tsx]: fixed.

**Review by rameshx — APPROVED** [review]

**Review by abeg777 — APPROVED** [review]

**Comment by knagurski** [comment]
Force merged due to lack of coverage in untouched code.

**Pull request merged**